ddd-koans/Value Object Pt. 2 & Entity/Collection Aversion/src/Tempo.kt

```diff
@@ -0,0 +1,18 @@
+import java.math.BigDecimal
+import java.time.Duration
+
+data class Tempo(val bpm: Int) {
+    init {
+        require(bpm in 1..399) { "Beat per minute values should be between zero and 400 (both exclusive)" }
+    }
+}
+
+data class Length(private val durationInSeconds: BigDecimal) {
+    val duration: Duration = Duration.ofMillis(durationInSeconds.multiply(1000.toBigDecimal()).toLong())
+
+    operator fun plus(other: Length) = Length(durationInSeconds.plus(other.durationInSeconds))
```

Comment:
just spotted that it's a good example of combinability of VOs - just wort keeping in mind when wirtting some descriptions to refer to sth that already exists and then ask user to do similar thing

Reply:
In the next task, I thought we can point out that example and then order to make sth similar for Notes. I especially tried to tell as little as possible about the method in the current task.